docs/proposals/PROPOSAL_F.md

```diff
@@ -0,0 +1,374 @@
+# Proposal F
+
+OCI Index references it all
+
+## Description
+
+This proposal cherry picks ideas from [Proposal C](https://github.com/opencontainers/wg-reference-types/blob/main/docs/proposals/PROPOSAL_C.md) and [Proposal D](https://github.com/opencontainers/wg-reference-types/blob/main/docs/proposals/PROPOSAL_D.md).
+A **nested** [OCI Index](https://github.com/opencontainers/image-spec/blob/main/image-index.md) (an OCI Index referencing another OCI index, allowed by the Image Specification) is used as a top-level object to reference all the artifacts attached to an image.
+No change to the reference is expected, but a **well-known annotations** section is added.
+
+## Links
+
+| Description    | Link                                                 |
+| -------------- | ---------------------------------------------------- |
+| OCI Artifacts  | [View](https://github.com/opencontainers/artifacts)  |
+| Image Spec     | [View](https://github.com/opencontainers/image-spec) |
+
+## Modifications
+
+### JSON Schema
+
+The image-spec json schema is left unchanged.
+
+The OCI Index object references OCI image manifests and OCI artifacts attached to them, using OCI descriptors.
+OCI Artifacts descriptors **SHOULD** contain **reference** annotations, to expose the reference type and the image referenced.
+OCI Manifest descriptors **SHOULD** contain **well-known** annotations, to expose metadata attached to the image.
+
+#### Pre-defined annotations
+
+Reference annotations are added to the [Annotations](https://github.com/opencontainers/image-spec/blob/main/annotations.md) section.
+* `org.opencontainers.reference.digest`: digest referencing an image. The digest **SHOULD** be referenced as OCI Descriptor in the root OCI index **OR** in the nested OCI index to avoid a weak reference.
+* `org.opencontainers.reference.type`: type of the artifact referenced by the descriptor.
+* `org.opencontainers.reference.description`: Human-readable description of the reference
+
+Example of a reference OCI descriptor, pointing to a SBOM referencing an alpine image on a specific platform:
+```json
+{
+   "mediaType": "application/vnd.cncf.oras.artifact.manifest.v1+json",
```

Comment:
We may want to pick a media type known to OCI here.

Reply:
Do you mean from [this list](https://github.com/opencontainers/image-spec/blob/main/media-types.md#oci-image-media-types) or that one needs to be defined for SBOMs specifically?

Reply:
That list looks right. For a manifest, anything other than a docker image, docker manifest list, OCI image, or OCI index is going to be rejected by most registries.

Reply:
Fixed https://github.com/opencontainers/wg-reference-types/pull/52/commits/47a0daa6a00ac31c25ba0d180c158ff9442e6634